### PR TITLE
Added examples to help.

### LINKED ISSUE #7: add better examples to functions

### PR DESCRIPTION
**Pull request by maju116**
Closes #7

### CODE REVIEW

File: R/dropdown_menu_output.R [hunk 1]
```diff
@@ -3,6 +3,13 @@
 #' @param outputId Id of the output.
 #' @return A dropdown menu that can be passed to \code{\link[semantic.dashboard]{dashboardHeader}}
 #' @export
+#' @examples
+#' dropdownMenuOutput("dropdown1")
+#'
+#' renderDropdownMenu({
```
Comment (filipstachura) [inline R/dropdown_menu_output.R]: This is not going to work, is it?

File: R/dropdown_menu_output.R [hunk 2]
```diff
@@ -18,6 +25,13 @@ dropdownMenuOutput <- dropdown_menu_output
 #' @param quoted Is expr a quoted expression (with \code{quote()})? This is useful if you want to save an expression in a variable.
 #' @return A dynamic dropdown menu that can be assigned to output.
 #' @export
+#' @examples
+#' dropdownMenuOutput("dropdown1")
+#'
+#' renderDropdownMenu({
```
Comment (filipstachura) [inline R/dropdown_menu_output.R]: ditto?

BTW `dropdown1` is not a perfect choice - you should remove 1 at the end to make the example as easy as possible.

File: R/menu_item.R [hunk 3]
```diff
@@ -75,6 +77,10 @@ menuSubItem <- menu_item
 #' @param ... Menu items.
 #' @return A sidebar menu that can be passed \code{\link[semantic.dashboard]{dashboardSidebar}}
 #' @export
+#' @examples
+#' sidebarMenu(
+#'   menuItem(tabName = "plot_tab", text = "My plot", icon = icon("home")),
+#'   menuItem(tabName = "table_tab", text = "My table", icon = icon("smile")))
```
Comment (filipstachura) [inline R/menu_item.R]: close sidebar in separate line

File: R/value_box_output.R [hunk 4]
```diff
@@ -4,6 +4,17 @@
 #' @param width Width of the valueBox.
 #' @return A value box that can be passed to \code{\link[semantic.dashboard]{dashboardBody}}
 #' @export
+#' @examples
+#' valueBoxOutput("value1")
```
Comment (filipstachura) [inline R/value_box_output.R]: I don't understand this kind of examples. What exactly does it show to the users?

File: R/dropdown_menu_output.R (hunk 2 above)
Comment (maju116) [reply R/dropdown_menu_output.R]: done

File: R/menu_item.R (hunk 3 above)
Comment (maju116) [reply R/menu_item.R]: done

File: R/dropdown_menu_output.R (hunk 1 above)
Comment (maju116) [reply R/dropdown_menu_output.R]: added \donotrun

**Pull request merged**